controllers/api/openstack/machines.py

```diff
@@ -17,6 +17,13 @@ class Machines(object):
     @cherrypy.tools.isAuthorised(redirect=True)
     @cherrypy.tools.jinja(template="machines/index.html")
     def index(self):
+
+        try:
+            username = cherrypy.request.cookie.get('fedid').value
+        except:
```

Comment:
Shouln't use bare except. What exception do you get if you can't fetch a cookie?